includes/class-wc-siftscience-admin.php

```diff
@@ -74,9 +74,16 @@ public function output_settings_fields() {
 			}
 		}
 
+		private function get_auto_text_style(){
```

Comment:
My preference is for this function to be located below the function that uses it.

Also, please add a space before the `{`. More information on WordPress coding standard can be found here: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style